docs(rocsparse): changelog entry for wave-aware SpMV/SpMM launch tuning (#10726)
ROCm/rocm-libraries: projects/rocsparse/CHANGELOG.md

* Added the `rocsparse_dnvec_descr_create_scalar` auxiliary routine, which creates a size-one dense vector descriptor for a host or device scalar.
* Added batched support to the SpMM algorithm `rocsparse_spmm_alg_csr_nnz_split`.

### Optimized

> * Architecture-aware launch-configuration tuning (launch-only, with no numerical or algorithmic changes) improves performance and performance-portability across RDNA (wave32) and CDNA (wave64) GPUs for several sparse level 2 and level 3 routines, including sparse matrix-vector multiplication (`rocsparse_spmv`) for the CSR adaptive, nnz-split, and LRB algorithms, the COO (SoA and AoS) formats, and the ELL format (`rocsparse_Xellmv`); the block-sparse `rocsparse_Xbsrmv` and `rocsparse_Xbsrxmv` routines; the `rocsparse_Xgemvi` and `rocsparse_Xgemmi` routines; and sparse matrix-matrix multiplication (`rocsparse_spmm`) for the blocked-ELL format.

Reviewer: ```suggestion
* Optimized architecture-aware launch configurations for RDNA (wave32) and CDNA (wave64) GPUs, improving performance and performance portability for several sparse level 2 and level 3 routines without algorithmic or numerical changes. Affected routines include `rocsparse_spmv` for the CSR adaptive, nnz-split, and LRB algorithms, the COO (SoA and AoS) formats, and the ELL format (`rocsparse_Xellmv`); `rocsparse_Xbsrmv`; `rocsparse_Xbsrxmv`; `rocsparse_Xgemvi`;  `rocsparse_Xgemmi`; and  `rocsparse_spmm`  with the blocked-ELL format.
```

Reviewer: Just trying to simplify things a bit.
